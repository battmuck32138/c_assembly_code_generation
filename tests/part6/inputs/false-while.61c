int main () {
  int x = -4;
  while (False) {
    if (x > 15) {
      break;
    }
    x = x * -2;
  }
  return x;
}
