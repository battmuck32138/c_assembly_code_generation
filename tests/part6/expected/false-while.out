
Exited with error code -4
